tests/integration/test_cli_integration.py

```
@@ -0,0 +1,442 @@
+        """Test list command with filters."""
+        mock_client = Mock()
+        mock_client_class.return_value = mock_client
+
+        mock_pr = Mock()
+        mock_pr.number = 1
+        mock_pr.title = "Filtered PR"
+        mock_pr.state = "open"
+        mock_pr.user.login = "alice"
+        mock_pr.labels = [Mock(name="bug")]
+
+        mock_repo.get_pulls.return_value = [mock_pr]
```

Ruff (0.13.1)

Undefined name `mock_repo`

(F821)